ckanext/cfpb_extrafields/plugin.py

```diff
@@ -1,43 +1,160 @@
+            'transfer_initial_size': [tk.get_validator('ignore_missing'),
+                                      v.positive_number_validator,
+                                      tk.get_converter('convert_to_extras')],
+            'transfer_method': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'also_known_as': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'content_periodicity': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'content_spatial': [tk.get_validator('ignore_missing'),
+                                v.input_value_validator,
+                        tk.get_converter('convert_to_extras')],
+            'update_frequency': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'website_name': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'website_url': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'wiki_link': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'dataset_notes': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'access_notes': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'acquisition_method': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'pra_exclusion': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'dataset_last_modified_date': [tk.get_validator('ignore_missing'),
+                                           v.reasonable_date_validator,
+                        tk.get_converter('convert_to_extras')],
+            'pra_omb_expiration_date': [tk.get_validator('ignore_missing'),
+                                           v.reasonable_date_validator,
+                        tk.get_converter('convert_to_extras')],
+            'privacy_has_privacy_act_statement': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'privacy_pia_notes': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'records_retention_schedule': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+            'procurement_document_id': [tk.get_validator('ignore_missing'),
+                        tk.get_converter('convert_to_extras')],
+
         })
-        # pprint.pprint(schema)
+        schema.update({
```

Comment:
The second update is for tag fields (convert_to_tags), so it's nice to have some indication that something different is happening. It's also just what the docs did.